### PR TITLE
[MANOPD-87147] Improving colors

### PR DESCRIPTION
**Pull request by akor0421**
### Description
TestCase and TestSuite classes from [testsuite.py](https://github.com/Netcracker/KubeMarine/blob/main/kubemarine/testsuite.py) always use colors to display text and background, even if `colorize=false` is specified in the arguments.

### Solution
Now, when calling TestCase, execution arguments are also passed, after which `colorize=false` is checked  in them for stdout.

Also now in Windows, colors are also used by default. And the small doc update about using Windows Terminal in Windows for correct color display.

### Test Cases
Run check_paas/check_iaas procedure with `colorize=false` for stdout
` kubemarine check_paas --log="stdout;level=warning;colorize=false"`

### Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Integration CI passed
- [ ] Unit tests. If Yes list of new/changed tests with brief description
- [x] There is no merge conflicts

### CODE REVIEW

**Review by n549 — APPROVED** [review]

**Comment by koryaga** [comment]
@OleksandrIaresko  pls validate

File: kubemarine/procedures/check_iaas.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 from kubemarine.core.group import NodeGroupResult
 
 def connection_ssh_connectivity(cluster):
-    with TestCase(cluster.context['testsuite'], '001', 'SSH', 'Connectivity', default_results='Connected'):
+    with TestCase(cluster.context['testsuite'], cluster.context['execution_arguments'], '001', 'SSH', 'Connectivity', default_results='Connected'):
```
Comment (ilia1243) [inline kubemarine/procedures/check_iaas.py:39]: Maybe we could pass the only property cluster.context to TestCase constructor, and fetch both properties `testsuite`, 'execution_arguments' from it there.

File: kubemarine/testsuite.py [hunk 2]
```diff
@@ -138,6 +154,18 @@ def get_summary(self, show_description=False, show_hint=False, show_minimal=Fals
...
+        if self.args.get('log') is not None:
+            for argsgroup in self.args.get('log'):
+                if argsgroup[0].split(";")[0] != 'stdout':
+                    continue
+                for argument in argsgroup[0].split(";"):
+                    if argument == 'colorize=false':
+                        return False
+            return True
+        else:
+            return True
```
Comment (ilia1243) [inline kubemarine/testsuite.py:167]: Alternatively, more simpler:
```suggestion
    def check_color(self):
        for handler in cluster.log.handlers:
            if isinstance(handler, log.StdoutHandler) and handler.formatter.colorize:
                return True
        return False
```
provided that either cluster, or cluster.log is passed to TestCase instance.

File: kubemarine/procedures/check_iaas.py (hunk 1 above)
Comment (akor0421) [reply kubemarine/procedures/check_iaas.py]: Done, passed `cluster` to TestCase to take `cluster.log` too

**Review by ilia1243 — APPROVED** [review]

**Pull request merged**